src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -171,7 +178,7 @@ impl SinkConfig for CloudwatchLogsSinkConfig {
 
         let client = self.create_client(cx.resolver())?;
         let svc = ServiceBuilder::new()
-            .concurrency_limit(request.in_flight_limit)
+            .concurrency_limit(request.in_flight_limit.unwrap())
```

Comment:
Since this could be `None` if the user configured "auto" for this sink, this should produce a less generic error than unwrap provides.

Reply:
IIRC `auto` should produce `Some(5)`? Anyway, I was planning to use  `if_none` after fixing test..

Reply:
Changed to `unwrap()` to `unwrap_or(5)`.

Reply:
The parsed `TowerRequestSettings::in_flight_limit` uses `Some(_)` for a fixed configured concurrency and `None` for `auto`. I should probably have made another enum like `InFlightLimit` (without `None`) there to make it explicit. This is already unwrapped such that an unconfigured limit maps to `Some(5)`.

To put it another way, this will let this work, but turns an explicitly configured `auto` into a fixed limit of 5 instead of producing an error. That doesn't seem to be ideal.

Reply:
Changed to `.expect`.